parent/pom.xml

```diff
@@ -473,6 +475,55 @@
                 <version>${micronaut.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.micronaut</groupId>
```

Comment:
So, what about using a `bom`?

Reply:
micronaut-bom downgrades <junit5.version>5.7.2</junit5.version> , but ignite 3 uses 5.8.1 version. It actually breaks the compilation. Are we ok to downgrade the junit version?

Reply:
I don't understand how this can happen. Shouldn't our explicit dependency override what is stated in the bom?

Reply:
junit-bom does not override this